### PR TITLE
Add a notebook on running romancal

### PR DESCRIPTION
**Pull request by nden**

### CODE REVIEW

File: general/Running the Roman pipeline.ipynb [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+    "\n",
+    "### Pipeline installation\n",
+    "\n",
+    "#### CRDS Setup\n",
+    "\n",
+    "\n",
+    "```\n",
+    "export CRDS_PATH=$HOME/crds_cache\n",
+    "export CRDS_SERVER_URL=https://roman-crds-test.stsci.edu\n",
+    "```\n",
```
Comment (sosey) [inline general/Running the Roman pipeline.ipynb:18]: do we want to mention here that the test server isn't public, but that this is the way to set it? Also mention that offline, the crds cache path is used by the installed crds client? and maybe that when the server is referenced, missing reference files that are required are downloaded and installed in the cache?

File: general/Running the Roman pipeline.ipynb [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "22bdae5d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from roman_datamodels import datamodels as rdm\n",
+    "\n",
+    "level1_file = \"/grp/roman/scsb/build_0.3/l1_0004_science_raw.asdf\"\n",
```
Comment (sosey) [inline general/Running the Roman pipeline.ipynb]: shall we make this for a general raw file that we make available in box?

File: general/Running the Roman pipeline.ipynb [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "99d57663",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "level2_file = \"/grp/roman/scsb/build_0.3/l2_0004_rate.asdf\""
```
Comment (sosey) [inline general/Running the Roman pipeline.ipynb]: same generalization comment as above for the input level 1 file

File: general/Running the Roman pipeline.ipynb [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+   "source": [
+    "result.save('my_flat_fielded_image.asdf')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "57b81320",
+   "metadata": {},
+   "source": [
+    "### Running a step with local reference files\n",
```
Comment (sosey) [inline general/Running the Roman pipeline.ipynb:212]: Do we want to also include an example of using the code in a python env directly? i.e. not calling stpipe, but importing the step?

**Review by sosey — CHANGES_REQUESTED** [review]

File: general/Running the Roman pipeline.ipynb (hunk 2 above)
Comment (nden) [reply general/Running the Roman pipeline.ipynb]: I updated the uncal file in Box with the latest version and added a generated L2 file.

File: general/Running the Roman pipeline.ipynb (hunk 3 above)
Comment (nden) [reply general/Running the Roman pipeline.ipynb]: next cell?

**Comment by sosey** [comment]
@nden  I had to move the location of the box folder, so we need to update the data links

**Comment by nden** [comment]
@sosey I believe I addressed all comments.

**Review by sosey — APPROVED** [review]

**Pull request merged**